docs/spec.json

```diff
@@ -2395,4 +2494,4 @@
       "in": "query"
     }
   }
-}
\ No newline at end of file
+}
```

Comment:
The travis ci build probably doesn't like this newline

Reply:
Hm, the Swagger specification check fails locally, too. I'm not sure, but the `x-go-package` in the specification seems to be generated wrongly to me. It starts with a `_` character, not with the `github.com` domain.

Reply:
Yeah, you can ignore these changes and just remove the newline here and the spec should generate successfully. I'm guessing that travis builds gotify in $GOPATH and thus has there packages with github.com.

Reply:
Here a successful build with your branch https://travis-ci.org/gotify/server/builds/651646654 with this change https://github.com/gotify/server/commit/138609f119d0f928fa5616b97440e2c9c0546a4e